src/TodoList.jsx

```diff
@@ -0,0 +1,22 @@
+
+function TodoList() {
+    const todos = [
+        { id: 1, title: "review resources" },
+        { id: 2, title: "take notes" },
+        { id: 3, title: "code out app" },
+    ]
+    {/*code continues...*/ }
+
+    {/*extract from App.jsx*/ }
+    {/*...code*/ }
+    return (
+        <div>
+            <h1>Todo List</h1>
```

Comment:
This `h1` belongs in the top level component. So you want this rendered in your `App` component.